docs/source/model_doc/gpt_neo.rst

```diff
@@ -10,22 +10,39 @@
     an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
     specific language governing permissions and limitations under the License.
 
-GPTNeo
+EleutherAI GPT Neo
 -----------------------------------------------------------------------------------------------------------------------
 
 Overview
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The GPTNeo model was proposed in `<INSERT PAPER NAME HERE> <<INSERT PAPER LINK HERE>>`__ by <INSERT AUTHORS HERE>.
-<INSERT SHORT SUMMARY HERE>
+The GPTNeo model was released in the `EleutherAI/gpt-neo <https://github.com/EleutherAI/gpt-neo>`__ repository by Sid
+black, and Stella Biderman. It is a GPT2 like causal language model trained on the `Pile <https://pile.eleuther.ai/>`__
```

Comment:
```suggestion
Black, and Stella Biderman. It is a GPT2 like causal language model trained on the `Pile <https://pile.eleuther.ai/>`__
```

Reply:
I believe @leogao2, @lucidrains and @ConnorJL also greatly contributed to that effort. Sid and team, who would you like for us to mention here?

Reply:
Yes, all three of them contributed much more than I.

Reply:
Yes, all three of them contributed much more than I and should absolutely be credited.